README.md

```diff
@@ -6,3 +6,21 @@
 ### generate go http client go bindings via openapi
 - use `./generate_gobindings.sh` to generate the golang bindings (for pinning services of pinata and web3.storage). 
 - There are some fixes you would need to do (missing braces etc.)
+
+### Run the server
+to start a server which listens for request on a particular port, run:
+```bash
+go run main.go -port 3000 -jwt "<jwt_token>"
```

Comment:
remove this file...this file is there in the https://github.com/covalenthq/ipfs-pinner/pull/6 PR